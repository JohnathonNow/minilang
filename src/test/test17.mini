with (A, B, C) := (1, 2, 3) do
	print('A = {A}, B = {B}, C = {C}\n')
end