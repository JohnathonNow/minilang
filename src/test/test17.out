A = 1, B = 2, C = 3
L[1] := 1
L[2] := 2
L[3] := 3
L[4] := 4
L[5] := 5
L[6] := 6
L[7] := 7
L[8] := 8
L[9] := 9
L[10] := 10
L[11] := 11
L[12] := 12
L[13] := 13
L[14] := 14
L[15] := 15
L[16] := 16
L[17] := 17
L[18] := 18
L[19] := 19
L[20] := 20
L[17] := 17
L[44] := 44
L[77] := 77
L[84] := 84
L[37] := 37
L[24] := 24
L[97] := 97
L[64] := 64
L[57] := 57
L[4] := 4
L[17] := 17
L[44] := 44
L[77] := 77
L[84] := 84
L[37] := 37
L[24] := 24
L[97] := 97
L[64] := 64
L[57] := 57
L[4] := 4
